torch/jit/_builtins.py

```diff
@@ -139,3 +139,49 @@ def _register_builtin(fn, op):
 
 def _find_builtin(fn):
     return _get_builtin_table().get(id(fn))
+
+
+# These are Python globals that have special sugaring in the compiler. See
+# [python globals] in the compiler for details. Adding a builtin global here
+# indicates that when resolving, we should skip these so they dispatch to
+# the correct sugared representation
+compiler_builtins = None
+def _is_global_builtin(maybe_builtin):
+    global compiler_builtins
+    if compiler_builtins is None:
+        # Errors (AssertionError and RuntimeError) should not be here since
+        # they are handled separately in the compiler
+        compiler_builtins = set([
```

Comment:
Can you replace using https://stackoverflow.com/questions/22864221/is-the-list-of-python-reserved-words-and-builtins-available-in-a-library ? Don't think it's necessary to enumerate them all.

Reply:
I ran into some problems with this, we support certain builtins as Python calls so they shouldn't be skipped here, we should only skip the ones that we have explicit support for inside the compiler, so we need them to be listed out.